INFO:hyena.py:84:2022-03-27 16:41:53,572:Loaded moderation
INFO:hyena.py:102:2022-03-27 16:41:56,969:Logged in as Mirage#6766 (ID: 870347922456346644)
INFO:hyena.py:103:2022-03-27 16:41:56,969:Initial: synced slash commands
ERROR:hyena.py:106:2022-03-27 16:42:10,612:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:11,044:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:11,699:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:12,368:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:12,991:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:13,475:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:16,581:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:17,163:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:17,674:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:18,187:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

ERROR:hyena.py:106:2022-03-27 16:42:18,699:Traceback (most recent call last):
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/venv/lib/python3.9/site-packages/discord/client.py", line 373, in _run_event
    await coro(*args, **kwargs)
  File "/Users/siddhant/Documents/Programming/Python/projects/hyena-hostable/src/cogs/moderation.py", line 49, in on_message
    blah
NameError: name 'blah' is not defined

